src/main.rs

```diff
@@ -0,0 +1,83 @@
+use mdbook::renderer::RenderContext;
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::io;
+
+#[derive(Deserialize)]
+struct I18nConfiguration {
+    languages: BTreeMap<String, String>,
+    default_language: Option<String>,
+
+    #[serde(default)]
+    translate_all_languages: bool,
+    #[serde(default)]
+    move_translations_to_html_directory: bool,
+}
+
+fn main() {
+    let mut stdin = io::stdin();
+
+    // Get the configs
+    let ctx = RenderContext::from_json(&mut stdin).unwrap();
+    let i18n_config: I18nConfiguration = ctx
+        .config
+        .get_deserialized_opt("output.i18n-helpers")
+        .unwrap()
+        .unwrap();
+
+    if !i18n_config.translate_all_languages {
+        return;
+    }
+
+    let mut mdbook = mdbook::MDBook::load(&ctx.root).expect("Failed to load book");
+    // Overwrite with current values from stdin.
```

Comment:
When I read this, I wonder how the values can be different? Perhaps explain why this is necessary/useful in the comment?

Reply:
I rewrote that part a bit. That's why we have `move_translations_to_html_directory` also